specification/app/resource-manager/Microsoft.App/stable/2022-05-01/examples/ManagedEnvironments_CreateOrUpdate.json

```diff
@@ -35,7 +40,14 @@
               "customerId": "string"
             }
           },
-          "zoneRedundant": true
+          "zoneRedundant": true,
+          "customDomainConfiguration": {
```

Comment:
Update example for get/list/patch

Reply:
Updated sample for get/list, seems like we don't support patch for environment now.